ballerina/Ballerina.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 org = "ballerina"
 name = "grpc"
-version = "1.5.0"
-distribution = "2201.2.0"
+version = "1.6.0"
+distribution = "2201.3.0-20221004-103700-89b78c16"
```

Comment:
Module version should be changed through gradle.properties and distribution through `build-config/resources/Ballerina.toml`